Source/WarpX.cpp

```diff
@@ -334,7 +334,19 @@ WarpX::ReadParameters ()
                "The boosted frame diagnostic currently only works if the boost is in the z direction.");
 
         pp.get("num_snapshots_lab", num_snapshots_lab);
-        pp.get("dt_snapshots_lab", dt_snapshots_lab);
+
+        // Read either dz_snapshots_lab or dt_snapshots_lab
+        bool snapshot_interval_is_specified = 0;
+        Real dz_snapshots_lab = 0;
+        snapshot_interval_is_specified += pp.query("dt_snapshots_lab", dt_snapshots_lab);
+        if ( pp.query("dz_snapshots_lab", dz_snapshots_lab) ){
+            dt_snapshots_lab = dz_snapshots_lab/moving_window_v;
```

Comment:
Is moving_window_v the correct velocity for this? What happens when it is zero?

Reply:
I think you're right, this should be `PhysConst::c`. It is arbitrary, but I think it makes sense to follow something that propagates at c when running in a boosted frame. I can modify that and add something in the doc.